Reading dataset /home/idrmal/RFC/resources/datasets/Iris/Iris.csv
Invalid line 152 : []
Traceback (most recent call last):
  File "/home/idrmal/RFC/rfc/prepare_dataset.py", line 227, in <module>
    process_dataset(word,1000,4,10)
  File "/home/idrmal/RFC/rfc/prepare_dataset.py", line 165, in process_dataset
    featureData = binning(featureData, continuousFeatureList, 10)
  File "/home/idrmal/RFC/rfc/prepare_dataset.py", line 143, in binning
    temporaryBucket[index].append(featureData[sample][continuousFeatureList[index]])
IndexError: list index out of range
